### ISSUE TITLE
Focussed Flagging Permission on "Report this review" 

### ISSUE THREAD

**Issue by erosman** [mozilla/addons#517]
I had already brought up the issue but since we had another talk about in on Mozfest, let us revisit the suggestion.

Currently, everyone can flag a review which can lead to review wars (flagging good reviews as spam) and unnecessary flagging. 

On the other hand, bug reports, although not meant for the review channel, could be missed by the developer when someone other than the developer flags them and they get deleted before the developer becomes aware of them.

The current options are:

> Please select a reason:
> 
>     Spam or otherwise non-review content
>     Inappropriate language/dialog
>     Misplaced bug report or support request
>     Other (please specify)

The proposal is to limit the user groups options e.g.:

- Spam or otherwise non-review content: to be available to developer/s only
- Inappropriate language/dialog: to be available to all
- Misplaced bug report or support request: to be available to developer/s only
- Other (please specify): to be available to all

In other words, only developers should be able to flag a review as Spam/Bug report/Support request

**Comment by jvillalobos** [comment]
Developers get notified about reviews by email, so if there's a valid bug being reported, they at least have access to that. The review being deleted only means they lose the chance to followup with the poster.

We have plans to better direct users to the right feedback channel in the near future, to mitigate misreported bugs. We should investigate if moderation reasons are useful at all before considering making them more complex by limiting to certain groups of people. I'm closing this for now.
